lib/worker.js

```diff
@@ -22,9 +22,6 @@ function Worker(queues, options) {
     // This worker will only process jobs of this priority or higher
     this.minPriority = options.minPriority;
 
-    if (this.queues.length === 0) {
```

Comment:
1) var worker = client.worker(); //Empty worker. Can be filled dinamically. (this.queues.length === 0  this.universal === false)

2) var worker = client.worker('*'); //Universal worker. Can't add new queues there (this.queues.length === 1 &&  this.universal === true)

May be you mean second variant ?